node/src/services/blockchain/events.rs

```diff
@@ -39,7 +40,7 @@ impl EventBusMessage for NewStorageRequest {}
 pub struct AcceptedBspVolunteer {
     pub who: AccountId32,
     pub location: FileLocation,
-    pub fingerprint: H256,
+    pub fingerprint: Key,
```

Comment:
While the underlying data is the same (a `Vec<u8>` representing an hash) the `fingerprint` is definitely not a `Key`. I think we need to rename `Key` to `FileKey` to make it clear.

Reply:
From what we discussed last week, it seemed like a good idea to use a straightforward `[u8; 32]` for the `fingerprint` type.

Reply:
After some discussion, we decided on created a `struct Fingerprint(H256)` for the `fingerprint`.